docs/tutorials/java/ssd_inference.md

```diff
@@ -0,0 +1,182 @@
+# Multi Object Detection using pre-trained SSD Model via Java Inference APIs
+
+This tutorial shows how to use MXNet Java Inference APIs to run inference on a pre-trained Single Shot Detection (SSD) Model.
+
+The SSD model is trained on the Pascal VOC 2012 dataset. The network is a SSD model built on Resnet50 as base network to extract image features. The model is trained to detect the following entities (classes): ['aeroplane', 'bicycle', 'bird', 'boat', 'bottle', 'bus', 'car', 'cat', 'chair', 'cow', 'diningtable', 'dog', 'horse', 'motorbike', 'person', 'pottedplant', 'sheep', 'sofa', 'train', 'tvmonitor']. For more details about the model, you can refer to the [MXNet SSD example](https://github.com/apache/incubator-mxnet/tree/master/example/ssd).
+
+## Pre-Requisites
+
+To complete this tutorial, we need : 
+* [MXNet Java Setup on IntelliJ IDEA](/java/mxnet_java_on_intellij.html) 
+* [wget](https://www.gnu.org/software/wget/) To download model artifacts 
+* SSD Model artifacts
+    * Use the following script to get the SSD Model files : 
+    ```bash
+      data_path=/tmp/resnet50_ssd
+      mkdir -p "$data_path"
+      wget https://s3.amazonaws.com/model-server/models/resnet50_ssd/resnet50_ssd_model-symbol.json -P $data_path
+      wget https://s3.amazonaws.com/model-server/models/resnet50_ssd/resnet50_ssd_model-0000.params -P $data_path
+      wget https://s3.amazonaws.com/model-server/models/resnet50_ssd/synset.txt -P $data_path
+    ```
+* Dataset  : A few sample images to run inference on.
+    * Use the following script to download sample images : 
+    ```bash
+      image_path=/tmp/resnet50_ssd/images
+      mkdir -p "$image_path"
+      cd $image_path
+      wget https://cloud.githubusercontent.com/assets/3307514/20012567/cbb60336-a27d-11e6-93ff-cbc3f09f5c9e.jpg -O dog.jpg
+      wget https://cloud.githubusercontent.com/assets/3307514/20012563/cbb41382-a27d-11e6-92a9-18dab4fd1ad3.jpg -O person.jpg
+    ``` 
+
+Alternately, you can get the entire SSD Model artifacts + images in one single script from the MXNet Repository by running [get_ssd_data.sh script](https://github.com/apache/incubator-mxnet/blob/master/scala-package/examples/scripts/infer/objectdetector/get_ssd_data.sh)  
+     
+## Time to code! 
+1. Following the [MXNet Java Setup on IntelliJ IDEA](/java/mxnet_java_on_intellij.html) tutorial, in the same project `JavaMXNet`, create a new empty class called : `ObjectDetectionTutorial.java`. 
```

Comment:
Can someone follow along with their IDE of choice instead?

Reply:
Goes with the same point that I made above about using another IDE.

Reply:
Done.